Symfony/CS/Tests/IntegrationTest.php

```diff
@@ -33,6 +33,6 @@ protected static function getFixturesDir()
      */
     protected static function getTempFile()
     {
-        return self::getFixturesDir().'/tmp/tmp.php';
+        return static::getFixturesDir().'/tmp/tmp.php';
```

Comment:
indeed this was not needed, reverted (this one was needed https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1585/files#diff-f387a0305e98fb7de24f5f8699ec1ec6L92)